### DISCUSSION TITLE
Details of SDG Members

### DISCUSSION THREAD

**Discussion by somesh202** [COPS-IITBHU/sdg-site#18] [General]
Please comment down your details in the given format:

- Name:
- Designation:
- Github_URL:
- Linkedin_URL:
- Any other social media/website:

**Comment by Arvind0302** [comment]
Arvind Yadav
Web Developer
https://github.com/Arvind0302
https://www.linkedin.com/in/arvind-yadav-88a8391bb/

**Comment by Manan-Rathi** [comment]
Manan Rathi
Web Developer
https://github.com/Manan-Rathi
https://www.linkedin.com/in/manan-rathi-08b5471ba

**Comment by tiger-yash** [comment]
Yash Raj
Web Developer
https://github.com/tiger-yash
https://www.linkedin.com/in/yash-raj-9aa877165/

**Comment by Divyansh013** [comment]
Divyansh Srivastava
Web Developer
https://github.com/Divyansh013
https://www.linkedin.com/in/divyansh-srivastava-856627208/

**Comment by aswiniip** [comment]
P Aswini Kumar
Web Developer
https://github.com/aswinidev
https://www.linkedin.com/in/p-aswini-kumar-9186b7219/

**Comment by devansh12b2** [comment]
Devansh Mishra
Web/Flutter Developer,
https://github.com/devansh12b2
https://www.linkedin.com/in/devansh-mishra-3a6138203/

**Comment by debarati-06** [comment]
Debarati Bandopadhyay
Web Developer
https://github.com/debarati-06
https://www.linkedin.com/in/debarati-bandopadhyay-079641222/

**Comment by eshaanagarwal** [comment]
Eshaan Agarwal
Web Developer
https://github.com/eshaanagarwal
https://www.linkedin.com/in/eshaanagarwal

**Comment by it-is-skywalkerl** [comment]
Sanskar Santosh Totla
Web Developer
https://github.com/it-is-skywalkerl
https://www.linkedin.com/in/sanskar-santosh-totla-427314200/

**Comment by mihir02badkur** [comment]
Mihir Kumar Badkur
Android Developer
https://github.com/mihir02badkur
https://www.linkedin.com/in/mihir-badkur-17139620a/

**Comment by RainaJain5** [comment]
Raina Jain
Android Developer
https://github.com/RainaJain5
https://www.linkedin.com/in/raina-jain-6b4b68143

**Comment by ayushigupta931** [comment]
Ayushi Gupta
Android Developer
https://github.com/ayushigupta931
https://www.linkedin.com/in/ayushi-gupta-125082212

**Comment by king-11** [comment]
Lakshya Singh
Web Developer
https://github.com/king-11
https://www.linkedin.com/in/lakshyasingh11/

**Comment by yash-yp** [comment]
Yash Prakash
Web Developer
https://github.com/yash2002109
https://www.linkedin.com/in/yash-prakash-yp

**Comment by try-catch-stack** [comment]
Danish Ahmed Mirza
Web Developer
https://github.com/try-catch-stack
https://www.linkedin.com/in/danish306/

**Comment by sakshi-2412** [comment]
Ashlesha Dixit
Web Developer
https://github.com/sakshi-2412
https://www.linkedin.com/in/ashlesha-dixit/

**Comment by JeeveshGarg** [comment]
Jeevesh Garg
Web Developer 
https://github.com/JeeveshGarg
https://www.linkedin.com/in/jeevesh-garg-2130b3203

**Comment by S-kus** [comment]
Saumya Kushwaha
Web Developer
https://github.com/S-kus
https://www.linkedin.com/in/saumya-kushwaha-00ba101ba

**Comment by jyshdewangan** [comment]
Jayesh Dewangan
Web Developer
https://github.com/jyshdewangan
www.linkedin.com/in/jyshdewangan

**Comment by IamEzio** [comment]
Anshuman Maurya
Web Developer
[https://github.com/IamEzio](https://github.com/IamEzio)
[https://www.linkedin.com/in/anshuman-maurya-69587a220](https://www.linkedin.com/in/anshuman-maurya-69587a220)

**Comment by soumyo123-prog** [comment]
Soumyajyoti Dey
Web Developer
https://github.com/soumyo123-prog
https://www.linkedin.com/in/soumyajyoti-dey-809500194/

**Comment by ankur12-1610** [comment]
Ankur Patil
Web Developer
Github: https://github.com/ankur12-1610
Linkedin: https://www.linkedin.com/in/ankur-patil-a112a3202
Twitter: https://twitter.com/ankurrap?t=5-ArmFfr8tz6mFG6XGDI5w&s=09

**Comment by Swayamshu** [comment]
Swayamshu Swain
Web Developer 
https://github.com/Swayamshu
https://www.linkedin.com/mwlite/in/swayamshu-swain-bb0227208

**Comment by shubhanshu02** [comment]
Shubhanshu Saxena
Web Developer
https://github.com/shubhanshu02
https://www.linkedin.com/in/shubhanshu-saxena/

**Comment by MohitSharma-21** [comment]
Mohit Kumar Sharma
Web Developer
https://github.com/MohitSharma-21
https://www.linkedin.com/in/mohit-sharma-5073181b9

**Comment by evapilotno17** [comment]
Aniruddh Patil
Web Developer
https://github.com/lelouch-of-the-code
https://www.linkedin.com/in/aniruddh-patil-089850206/

**Comment by literalEval** [comment]
- Ravidev Pandey
- Android / Desktop Developer
- https://github.com/Literal-Eval
- https://www.linkedin.com/in/ravidev-pandey-07b650227

**Comment by shubhangi013** [comment]
Shubhangi
Web developer
https://github.com/shubhangi013 
https://www.linkedin.com/in/shubhangi-choudhary-32a347205

**Comment by atpug22** [comment]
Aryaman Gupta
Web Developer
https://github.com/atpug22
https://www.linkedin.com/in/ryaman/

**Comment by Hitansh-Shah** [comment]
Hitansh Shah
Web Developer
https://github.com/Hitansh-Shah
https://www.linkedin.com/in/hitansh-shah-b71161200

**Comment by AG-7219** [comment]
Ayush Gangwani
Web developer
https://github.com/AG-7219
https://www.linkedin.com/in/ayush-gangwani-1a277720b

**Comment by Aadeesh11** [comment]
Aadeesh Oswal
Flutter/Web developer
https://github.com/Aadeesh11
https://in.linkedin.com/in/aadeesh-oswal-b422941b9

**Comment by Tarun-Arora** [comment]
Tarun Arora
Web Developer
https://github.com/Tarun-Arora
https://www.linkedin.com/in/tarun-arora-25a260214/

**Comment by KKVANONYMOUS** [comment]
Kunal Kumar Verma
Web Developer
https://github.com/kkvanonymous
https://www.linkedin.com/in/kkvanonymous

**Comment by atharvabhatt200** [comment]
Atharva Bhatt
Web Developer
https://github.com/atharvabhatt200
https://www.linkedin.com/in/atharva-bhatt-199665156/

**Comment by chirayumit001** [comment]
Chirayu Mittal
Web Developer
https://github.com/chirayumit001
https://www.linkedin.com/in/chirayu-mittal-b6279a201/

**Comment by shukapurv** [comment]
Apoorve shukla
Web Developer
https://github.com/shukapurv
https://www.linkedin.com/in/apoorve-shukla-792b761ba

**Comment by dahiya-code** [comment]
Navya Dahiya
Web Developer
https://github.com/dahiya-code
https://www.linkedin.com/in/navyadahiya/

**Comment by Sriraj-dev** [comment]
Sriraj
App developer(Flutter)
https://github.com/Sriraj-dev
https://www.linkedin.com/in/sriraj-palakurthi-ab86061bb/
